docs/components/dropdown-menu.md

```diff
@@ -0,0 +1,394 @@
+    <KDropdownItem
+      :selected="selectedItem === 'us'"
+      @click="clickHandler('US selected', 'us', 'US (United States)')"
+    >
+      US (United States)
+    </KDropdownItem>
+    <KDropdownItem
+      :selected="selectedItem === 'fr'"
+      @click="clickHandler('France selected', 'fr', 'FR (France)')"
+    >
+      FR (France)
+    </KDropdownItem>
+  </template>
+</KDropdownMenu>
+
+<script>
+export default {
+  data() {
+    return {
+      selectedItem: '',
+      selectedLabel: 'Select an item'
+    }
+  },
+  methods: {
+    clickHandler (msg, val, label) {
+      if (val !== undefined) {
+        this.selectedItem = val
+      }
+
+      if (label) {
+        this.selectedLabel = label
+      }
+
+      this.$toaster.open(msg)
+    }
+  }
+}
+</script>
+```
+
+### showCaret
+
+Use this prop if you would like the trigger button to display the caret.
+
+<KDropdownMenu label="Documentation" :items="deepClone(defaultItemsUnselected)" show-caret />
+
+```html
+<KDropdownMenu
+  label="Documentation"
+  :items="items"
+  show-caret
+/>
+```
+
+### kpopAttributes
+
+Use the `kpopAttributes` prop to configure the **KPop** [props](/components/popover.html) dropdown.
+
+<div>
+  <KCard
+    title="Card Title"
+    body="Body Content"
+  >
+    <template #actions>
+      <KDropdownMenu
+        :kpop-attributes="{
+          popoverClasses: 'mt-5',
+          width: '180'
+        }"
+        :items="deepClone(defaultItemsUnselected)"
+      >
+        <template #default>
+          <KButton
+            appearance="secondary"
+            size="small"
+            class="non-visual-button"
+          >
+            <template #icon>
+              <KIcon
+                icon="more"
+                color="var(--black-400)"
+                size="16"
+              />
+            </template>
+          </KButton>
+        </template>
+      </KDropdownMenu>
+    </template>
+  </KCard>
+</div>
+
+```html
+<KCard
+  title="Card Title"
+  body="Body Content"
+>
+  <template #actions>
+    <KDropdownMenu
+      :kpop-attributes="{
+        popoverClasses: 'mt-5',
+        width: '180'
+      }"
+      :items="deepClone(defaultItemsUnselected)"
+    >
+      <template #default>
+        <KButton
+          appearance="secondary"
+          size="small"
+          class="non-visual-button"
+        >
+          <template #icon>
+            <KIcon
+              icon="more"
+              color="var(--black-400)"
+              size="16"
+            />
+          </template>
+        </KButton>
+      </template>
+    </KDropdownMenu>
+  </template>
+</KCard>
+```
+
+### disabled
+
+Use this prop to disable the dropdown, can be used in conjunction with `disabledTooltip` prop.
+
+### disabledTooltip
+
+Text to display on hover if dropdown is disabled.
+
+<div>
+  <KDropdownMenu
+    label="Documentation"
+    :disabled="true"
+    disabled-tooltip="You can't click me"
+    :items="deepClone(defaultItemsUnselected)"
+  />
+</div>
```

Comment:
issue: I can hover/click on the dropdown, the tooltip shows. If I then click on the page, the dropdown itself shows.

Reply:
The width of the element itself is too large and allows you to click (even not on the button) 
![image](https://user-images.githubusercontent.com/2229946/182469745-a170ce3d-9a2f-4f6a-8d09-4188eafe9ea6.png)

Reply:
Fixed